tooling/specs/specs.go

```diff
@@ -0,0 +1,72 @@
+package specs
+
+import "fmt"
+
+type maturity string
+
+const (
+	wip        maturity = "wip"
+	draft      maturity = "draft"
+	reliable   maturity = "reliable"
+	stable     maturity = "stable"
+	permanent  maturity = "permanent"
+	deprecated maturity = "deprecated"
```

Comment:
The updated specs specify maturity level per spec: https://github.com/ipfs/specs/pull/382/files#diff-68949a04625be3a1b31a438f35a6e8daaeb881ffd4a1631e083b3b070222e2f9